### PR TITLE
Roole tech added

### LINKED ISSUE #236: Add roole tech to build css
See http://roole.org/

### PR DESCRIPTION
**Pull request by narqo**
Closes #236 

\cc @dfilatov @mishanga

### CODE REVIEW

**Comment by mishanga** [comment]
Looks good. Tried on some examples, it works.

File: .bem/techs/roole.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        };
+    },
+
+    compileBuildResult : function(res) {
+        var defer = Q.defer();
+
+        ROOLE.compile(res, {
+            filename : './',
+            out : './',
+            prefixes : false
```
Comment (mishanga) [inline .bem/techs/roole.js]: It doesn't work. Write `box-shadow` to input.roo and check.
Use an empty array here.

Comment (narqo) [inline .bem/techs/roole.js]: curvedmark/roole#131

Ok

Comment (mishanga) [inline .bem/techs/roole.js]: :+1:

**Comment by narqo** [comment]
I'll `squash` all this stuff before merge, just ping me, that all is fine ;)

**Comment by mishanga** [comment]
No more bugs detected. Ship it!

**Pull request merged**